apisix/plugins/basic-auth.lua

```diff
@@ -161,6 +170,11 @@ function _M.rewrite(conf, ctx)
         return 401, { message = "Password is error" }
     end
 
+    -- 5. hide `Authentication` header if `hide_auth_header` is `true`
+    if conf.hide_auth_header == true then
+        core.response.set_header("Authentication", "")
```

Comment:
It seems you misunderstand the original issue. We want to hide the request header.

Reply:
My fault.  Done.